EssentialCSharp.Web/wwwroot/sitemap.xml

```diff
@@ -0,0 +1,939 @@
+<?xml version="1.0" encoding="utf-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <url>
+    <loc>https://essentialcsharp.com</loc>
+    <lastmod>2023-03-10T22:16:59.2615379+00:00</lastmod>
+    <changefreq>daily</changefreq>
+    <priority>1.0</priority>
+  </url>
+  <url>
+    <loc>https://essentialcsharp.com/TermsOfService</loc>
+    <lastmod>2023-03-10T22:16:59.2615379+00:00</lastmod>
```

Comment:
Let's not check this in

Reply:
I added an ignore for this file in the .gitignore on main so after a rebase and dropping this file it shouldn't appear

Reply:
Sounds good. I presume then its generated on deploy?

Reply:
Yes, it should be